src/functionaltests/java/com/ericsson/ei/subscriptions/trigger/SubscriptionTriggerSteps.java

```diff
@@ -153,10 +154,19 @@ public void check_mail_subscriptions_were_triggered() throws Throwable {
     @Then("^Rest subscriptions were triggered$")
     public void check_rest_subscriptions_were_triggered() throws Throwable {
         LOGGER.debug("Verifying REST requests.");
-        assert (requestBodyContainsStatedValues(REST_ENDPOINT));
-        assert (requestBodyContainsStatedValues(REST_ENDPOINT_AUTH));
-        assert (requestBodyContainsStatedValues(REST_ENDPOINT_PARAMS));
-        assert (requestBodyContainsStatedValues(REST_ENDPOINT_AUTH_PARAMS));
+        List<String> endpointsToCheck = new ArrayList<String>() {
```

Comment:
make it shorter 
List< String > endpointsToCheck = new ArrayList<>(Arrays.asList(REST_ENDPOINT,REST_ENDPOINT_AUTH,.....))

Reply:
Done